### PR TITLE
add setup using vagrant and virtualbox

### PR DESCRIPTION
**Pull request by komuW**
Hi,

- I have added config files on how to setup wifiphiser using vagrant and virtualbox.          
- Take a look and let me know if you need any other clarification.        

Thanks,          
Komu W.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,14 +36,30 @@ From the victim's perspective, the attack makes use in three phases:
...
+
 ## Help needed
 If you are a Python developer or a web designer you can help us improve wifiphisher. Feel free to take a look at the bug tracker for some tasks to do.
 
 ## Credits
 The script is based on an idea from <a
 href="https://github.com/DanMcInerney">Dan McInerney</a>. The parts for the
-jamming and selecting an AP have also been taken from his scripts <a
-href="https://github.com/DanMcInerney/wifijammer">wifijammer</a> and <a
+jamming and selecting an AP have also been taken from his scripts <ahref="https://github.com/DanMcInerney/wifijammer">wifijammer</a> and <a
```
Comment (sophron) [inline README.md]: This breaks the link here.

**Comment by sophron** [comment]
I'll test this as soon as i can.

File: README.md [hunk 2]
```diff
@@ -36,14 +36,30 @@ From the victim's perspective, the attack makes use in three phases:
...
+* clone this repository and change into its directory     
+* then execute the following commands from your terminal          
+`vagrant up`            
+`vagrant ssh`          
+`cd /vagrant`           
+* fix some path issues, ie, from inside the virtual machine:        
+`echo 'export PATH=$PATH:/usr/local/sbin' >> ~/.bashrc`            
+`echo 'export PATH=$PATH:/usr/sbin' >> ~/.bashrc`            
+`echo 'export PATH=$PATH:/sbin' >> ~/.bashrc`                   
+`source ~/.bashrc`
```
Comment (sophron) [inline README.md]: The backticks need to be removed here for the command to take effect.

**Comment by sophron** [comment]
Hi @komuW,

I was able to create the VM successfully but the wireless interfaces from the host machine were not bridged to the VM. This makes the script useless as it needs at least two wireless interfaces to work. Can you please fix this?

**Comment by komuW** [comment]
Hi @sophron , 
Thanks for taking a look at this. I will work on a fix and let you know. Thanks once again for having tested it out.

**Comment by blackHatMonkey** [comment]
I'm going to close this PR because of the inactivity. You are more than welcome to make another PR when you are ready.

**Pull request closed without merging**